### DISCUSSION TITLE
loop once

### DISCUSSION THREAD

**Discussion by Marcioghiraldelli** [AzuraCast/AzuraCast#7506] [General]
Hello, how are you?

I sometimes use the once loop when scheduling, but I still don't understand how it works.

If I schedule from 8 to 9, but my playlist is only 30 minutes long, it won't repeat, is that it? But it will start the next day at the same time, right?

**Comment by silvaalex2021** [comment]
Exactly. When scheduling a playlist, for example with a start time of 08:00 and an end time of 09:00, if the playlist is less than one hour long, AzuraCast may repeat the playlist or parts of it until the end time, 09:00. This option prevents that from happening.

**Reply by Heimatradio** [reply]
please write in english

**Reply by silvaalex2021** [reply]
Oops, sorry! My mistake. Since the person who started the discussion is from my country, I thought it'll be okay to give the answer directly in Portuguese, but I edited the comment already.
